src/main/resources/schemas/default.xml

```diff
@@ -161,9 +161,21 @@
     <tag name="EPILOGUE" depreciated="Use BACKMATTER with DIV instead of EPILOGUE" where="primary">
       <desc><![CDATA[]]></desc>
     </tag>
-    <tag name="FONTGROUP" where="oldspelling">
+    <tag name="FONT" where="oldspelling">
       <desc><![CDATA[This tag allows the editor to fine-tune font size where necessary, as, for example, on a title page.]]></desc>
-      <attribute name="n" type="select" default="3">
```

Comment:
Should the attribute be depreciated before it's dropped? If you can change them globally, and get the editors to make the change too, then it's moot.